app.log

```diff
@@ -0,0 +1,20 @@
+{"level":"info","message":"Fetching OpenAPI schema from: https://raw.githubusercontent.com/VilnaCRM-Org/user-service/main/.github/openapi-spec/spec.yaml...","timestamp":"2025-03-26T18:04:37.616Z"}
```

Comment:
Please do not commit the logs to the git
This file should be removed from git and then added to the .gitignore file